README.md

```diff
@@ -1,14 +1,103 @@
-# xCell
+![](https://raw.githubusercontent.com/JaimeRZP/Cosmoteka_tutorials/master/docs/src/assets/cosmoteka_logo.png)
 [![Coverage Status](https://coveralls.io/repos/github/xC-ell/xCell/badge.svg?branch=master)](https://coveralls.io/github/xC-ell/xCell?branch=master)
 
-Pipeline to compute Nx2pt angular power spectra and their covariances.
+Welcome to the Cosmoteka, the largest repository of consistently combined angular power spectra in Cosmology! 
 
-# Usage
-In order to run the code use `python3 run_cls.py input/kv450_1024.yml cls`.
```

Comment:
The link to the tutorials is good, but I'd leave a example of how to run this in the terminal. In the end, the idea is to run this stuff in parallel in a cluster, so better leave a comment on this.

Reply:
Brought it back!